tools/applicationinsights-web-snippet/README.md

```diff
@@ -0,0 +1,35 @@
+# Microsoft Application Insights JavaScript SDK - Web Snippet
+
+[![Build Status](https://travis-ci.org/microsoft/ApplicationInsights-JS.svg?branch=master)](https://travis-ci.org/microsoft/ApplicationInsights-JS)
+[![npm version](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-web-snippet.svg)](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-web-snippet)
+
+Web Snippet for the Application Insights Javascript SDK
+
+This project exists to publish latest Application Insights Javascript Web Snippet.
+
+
+
+## Build:
```

Comment:
It would be worth adding a section on how we expect people to use this and what the exported values mean etc.

ie. something about once they grab the value they will need to replace the instrumentation key / connection string section etc.